typings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can get rid of this when we move to `@dojo/interfaces/cli` and using the types from `@types/yargs`